pkg/virt-operator/install-strategy/create.go

```diff
@@ -1509,6 +1509,61 @@ func addOrRemoveSSC(targetStrategy *InstallStrategy,
 	return nil
 }
 
+func syncKubevirtNamespaceLabels(kv *v1.KubeVirt,
+	stores util.Stores,
+	clientset kubecli.KubevirtClient,
+	expectations *util.Expectations) error {
```

Comment:
expectations passed but not used in the function

Reply:
Nice catch. Fixed :+1: